### PR TITLE
[ES] Update `Gameplay/Grade`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`SKIP_WIKILINK_CHECK`

### CODE REVIEW

File: wiki/Gameplay/Accuracy/es.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ En osu!, la precisión se calcula ponderando el [juicio](/wiki/Gameplay/Judgemen
...
 
 ```
-300 -> 300 / 300 = 1   = 100.00%
-100 -> 100 / 300 = 1/3 =  33.33%
-50  ->  50 / 300 = 1/6 =  16.67%
-0   ->   0 / 300 = 0   =   0.00%
+300 -> 300 / 300 = 1   = 100.00 %
+100 -> 100 / 300 = 1/3 =  33.33 %
+50  ->  50 / 300 = 1/6 =  16.67 %
+0   ->   0 / 300 = 0   =   0.0 0%
```
Comment (agatemosu) [inline wiki/Gameplay/Accuracy/es.md:23]:
```suggestion
0   ->   0 / 300 = 0   =   0.00 %
```

File: wiki/Gameplay/Accuracy/es.md [hunk 2]
```diff
@@ -17,17 +17,17 @@ En osu!, la precisión se calcula ponderando el [juicio](/wiki/Gameplay/Judgemen
...
+0   ->   0 / 300 = 0   =   0.00 %
 ```
 
 ### ![](/wiki/shared/mode/taiko.png) osu!taiko
 
 ![Precisión = (número de GREATs + 0.5 \* número de GOODs) / (número de GREATs + número de GOODs + número de fallos)](img/accuracy_taiko_updated.png "Fórmula de precisión para osu!taiko")
 
-En osu!taiko, la precisión se calcula tomando la suma de la precisión de la nota (qué tan cerca estuvo de tocar la nota a tiempo) dividida por el número total de notas anotadas hasta el momento. Las precisiones de las notas están etiquetadas como GREAT (良) (cuenta como 100%), GOOD (可) (cuenta como 50%) (mitad), y MISS/BAD (不可) (cuenta como 0%, lo que también rompe el combo). Los redobles de tambor y los spinners no influyen en la precisión.
+En osu!taiko, la precisión se calcula tomando la suma de la precisión de la nota (qué tan cerca estuvo de tocar la nota a tiempo) dividida por el número total de notas anotadas hasta el momento. Las precisiones de las notas están etiquetadas como GREAT (良) (cuenta como 100 %), GOOD (可) (cuenta como 50 %) (mitad), y MISS/BAD (不可) (cuenta como 0 %, lo que también rompe el combo). Los redobles de tambor y los spinners no influyen en la precisión.
```
Comment (juankristal) [inline wiki/Gameplay/Accuracy/es.md:30]: No debería ser la precisión de `cada` nota en lugar de `la` nota?

File: wiki/Gameplay/Grade/es.md [hunk 3]
```diff
@@ -1,55 +1,64 @@
...
+  - xh
+  - rango
+  - s plateada
+  - ss plateada
 ---
 
-# Rangos
+# Grado
+
+*Para otros usos, véase [Rango (desambiguación)](/wiki/Disambiguation/Rank).*
```
Comment (juankristal) [inline wiki/Gameplay/Grade/es.md:16]: Tiene sentido esta aclaracion si cambiamos la nomenclatura?

File: wiki/Gameplay/Accuracy/es.md (hunk 2 above)
Comment (agatemosu) [reply wiki/Gameplay/Accuracy/es.md:30]: In English it says "the note accuracy"

File: wiki/Gameplay/Grade/es.md (hunk 3 above)
Comment (juankristal) [reply wiki/Gameplay/Grade/es.md:16]: I know that in english it does make sense, but for spanish it doesn't really as the names are different now.

**Review by juankristal — APPROVED** [review]

**Pull request merged**